crates/core/thin_edge_json/src/serialize.rs

```
@@ -92,13 +91,18 @@ impl Default for ThinEdgeJsonSerializer {
 impl MeasurementVisitor for ThinEdgeJsonSerializer {
     type Error = ThinEdgeJsonSerializationError;
 
-    fn visit_timestamp(&mut self, timestamp: DateTime<FixedOffset>) -> Result<(), Self::Error> {
+    fn visit_timestamp(&mut self, timestamp: OffsetDateTime) -> Result<(), Self::Error> {
         if self.is_within_group {
             return Err(MeasurementStreamError::UnexpectedTimestamp.into());
         }
 
         self.json.write_key("time")?;
-        self.json.write_str(timestamp.to_rfc3339().as_str())?;
+        self.json.write_str(
+            timestamp
+                .format(&format_description::well_known::Rfc3339)
+                .unwrap()
```

Unwrap? Shouldn't we propagate the error up to the caller?